scraper_root/scraper/bybitderivatives.py

```diff
@@ -144,10 +143,20 @@ def sync_open_orders(self):
                                 order.quantity = float(item['qty'])
                                 order.side = item['side'].upper()  # upper() to make it the same as binance
                                 # bybit has no 'position side', assuming 'side'
```

Comment:
I found no positionIdx on bybit.
There is "side" and this can be "buy" or "sell" and with "reduce_only" you can find out if it is long or short trade.
First implementation from passivebot uses the "order_link_id" for long or short, but this only works for trades initiated from passivebot, because this is a free text you can set when opening trades.

Reply:
> bybit's api uses positionIdx, and for a short the quantity is negative I believe?

- Position idx, used to identify positions in different position modes
0: One-Way Mode
1: Buy side of both side mode
2: Sell side of both side mode

Reply:
In the API for "Get Active Order" (https://bybit-exchange.github.io/docs-legacy/futuresV2/linear/#t-getactive) there is no position_idx. position_idx is only available for "My Position" (https://bybit-exchange.github.io/docs-legacy/futuresV2/linear/#t-myposition). Here in the code of the scraper we ask for the open orders, not for the open positions. I did not write this scraper, just trying to fix the errors. With my code the open orders side long/short is correct identified. With the original code, it is wrong. Some open long orders are countet as short and vice versa.